Monika After Story/game/script-topics.rpy

```diff
@@ -10093,12 +10093,12 @@ init 5 python:
 
 label monika_pygmalion:
     m 1eua "Hey [player], have you ever read {i}The Metamorphoses{/i}?"
-    m 3eua "It’s a collection of poems by an ancient Roman poet named Ovid."
-    m 1eud "There’s one poem that really struck me, called {i}The Story of Pygmalion and the Statue.{/i}"
+    m 3eua "It's a collection of poems by an ancient Roman poet named Ovid."
+    m 1eud "There's one poem that really struck me, called {i}The Story of Pygmalion and the Statue.{/i}"
     m 1eua "It tells the story of a sculptor, Pygmalion, who fell in love with an ivory statue that he had sculpted."
     m 3eua "He loved the statue so much, that during one of the festivals held for Aphrodite, he made an offering to the goddess and wished for her to grant life to the ivory statue."
     m 1eub "When he returned to his home, he kissed the statue and to his surprise, the lips of the statue felt warm."
-    m 1eua "He kissed the statue once more, and the ivory of statue lost its hardness, becoming soft like skin."
+    m 1eua "He kissed the statue once more, and the ivory lost its hardness, becoming soft like skin."
```

Comment:
Could we change this to "He kissed the statue once more, and the ivory _of the statue_ lost its hardness, becoming soft like skin."?

Reply:
I considered that option as well, but I think that adding "of the statue" doesn't really contribute anything meaningful to the line, as we already know we're talking about a statue.

Reply:
In terms of flow,I think it works better. Would rather that change personally

Reply:
Discussed this @multimokia and @jmwall24, they approve of my change.